cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -83,6 +83,8 @@ public static void main(final String[] args) {
                 type = args[++i];
             } else if (args[i].equals("--tool")) {
                 tool =  args[++i];
+            } else if (args[i].equals("--signSBOM")) {
```

Comment:
which flag exactly you want to use` --signSBOM `or `--sign`
they should be consistent when calling `"${javaHome}"/bin/java -cp "${classpath}" temurin.sbom.TemurinGenSBOM --sign <sbom.jsons>`